modules/ai/module.php

```diff
@@ -160,6 +162,36 @@ private function register_layout_experiment() {
 		] );
 	}
 
+	public function enqueue_ai_media_library() {
+		wp_enqueue_script( 'elementor-ai-media-library',
+			$this->get_js_assets_url( 'ai-media-library' ),
+			[
+				'jquery',
+				'elementor-v2-ui',
+				'elementor-v2-icons',
+				'media-grid',
+			],
+			ELEMENTOR_VERSION,
+			true
+		);
+
+		$session_id = 'elementor-editor-session-' . Utils::generate_random_string();
```

Comment:
Please create a different prefix for a session from `upload.php`: ` wp-media-library-session-`

```suggestion
		$session_id = 'elementor-editor-session-' . Utils::generate_random_string();
```